front-end/src/app/login/security-notice/security-notice.component.html

```diff
@@ -62,29 +63,27 @@ <h1>Security notification</h1>
               label="You consent to the security notification for one year"
               ariaLabel="You consent to the security notification for one year"
             ></p-checkbox>
-            <div class="col-12">
-              <app-error-messages
-                [form]="form"
-                fieldName="security-consent-annual"
-                [formSubmitted]="formSubmitted"
-              ></app-error-messages>
-            </div>
           </div>
         </form>
         <div class="align-center">
-          <p-button class="security-notice-button" label="Decline and exit" (click)="loginService.logOut()"></p-button>
           <button
```

Comment:
These buttons need to be updated to match the wireframe.  hopefully we already have an example of that teal secondary and navy primary

Reply:
updated checkbox to match wireframe.  Also the wireframe was updated to match the requirements (and what was implemented).